### DISCUSSION TITLE
这个项目我居然接手快一年了

### DISCUSSION THREAD

**Discussion by Arondight** [Arondight/Adachi-BOT#733] [General]
最开始 fork 的原因看来是原作者写的命令不好敲 5d7832c705d5187513031b09b7c6a8a6103f0ab6 …… 之后我也没啥修改，可能加了个“吃什么”，莫名项目开始长星星，应该是有了十几个🌟。然后没过多久作者为了重构项目，规范了不透明的资源 API ，于是在我照着原项目新版本的代码还原了最初版本的资源同步脚本 ae3f11f4de4e77e976fd90c3daca296cd468c0f7 之后这个项目独立了出来……

**Comment by Arondight** [comment]
5/13 到时候正好可以发个毫无卵用的版本，不知道新地图开了没，屯了两百抽就等草萝莉了

**Comment by mark9804** [comment]
题外话，我想不到晚饭做什么的时候真会靠 bot 决定

**Reply by Arondight** [reply]
现在 bot 的菜单，有很多我这辈子都没见过……

> 题外话，对我来说还是第一版的菜单劲大 https://github.com/Arondight/Adachi-BOT/commit/2f6d6255cd604c5ecdbb332899516e2ffa01b33b

**Reply by mark9804** [reply]
我塞了两季舌尖上的中国菜谱进去

**Comment by mark9804** [comment]
老大哥正好 1000 commits
<img width="451" alt="image" src="https://user-images.githubusercontent.com/9006264/162197933-ecdadc33-e712-46b5-b5e7-e795b84da6d1.png">

**Comment by Arondight** [comment]
回头一看，这个项目是真的粗糙，但是已经没法改了，改了就会波及到 fork ，另外我也懒得改了

**Reply by mark9804** [reply]
~~事到如今改动的地方还少吗~~

**Reply by Arondight** [reply]
都是小改，没有动筋骨过
